L1Trigger/TrackFindingTracklet/interface/TrackletProcessorDisplaced.h

```diff
@@ -43,14 +43,11 @@ namespace trklet {
     int iAllStub_;
     unsigned int maxStep_;
     int count_;
-    unsigned int layerdisk_;
 
-    int layer1_;
-    int layer2_;
-    int layer3_;
-    int disk1_;
-    int disk2_;
-    int disk3_;
+    unsigned int iSeed_;
```

Comment:
@cgsavard Since `TrackletProcessDisplaced` inherits from `TrackletCalculatorDisplaced`, and the latter already has an `iSeed_` member, this line shouldn't be necessary.
```suggestion
```